### DISCUSSION TITLE
Connecting DBeaver with Tableau

### DISCUSSION THREAD

**Discussion by bobthebuilder1776** [dbeaver/dbeaver#21369] [DBeaver Development]
Hello,

I am looking to integrate DBeaver and my PostgreSQL DBA with Tableau, but am running into a few issues. I tried following what DBeaver had on their website (https://dbeaver.com/docs/dbeaver/Tableau-integration-in-DBeaver/), but am even just having trouble filling the profile for Tableau connection. It says after general there should be a tableau button filter, but I have no such thing. Was hoping someone could maybe help me navigate this to connect DBeaver with my Tableau account. 
![image](https://github.com/dbeaver/dbeaver/assets/146376651/d9c5e49c-0fbb-4988-9cea-aa0723b89321)

**Comment by LonwoLonwo** [comment]
Hello @bobthebuilder1776 

This integration is supported only for our PRO products.
You can try it for free: https://dbeaver.com/trial/
